apps/console/src/index.tsx

```diff
@@ -36,6 +37,32 @@ import Theme from "./theme";
 // Set the runtime config in the context.
 ContextUtils.setRuntimeConfig(Config.getDeploymentConfig());
 
+// Function to check the status of the Monaco CDN.
+// If the CDN is not available, the default CDN will be used.
+async function checkCDNStatus() {
+    try {
+        const response: Response = await fetch("https://cdn.jsdelivr.net/npm/monaco-editor@0.46.0/min/vs/loader.js");
```

Comment:
Can't we do something like keeping a local copy of monaco editor in the `public` folder and referencing to it from here?

Reply:
Refer comment below